build/BuildLinuxTask.cs

```diff
@@ -9,24 +9,81 @@ public sealed class BuildLinuxTask : FrostingTask<BuildContext>
+        context.StartProcess(shellCommandPath, processSettings);
+        processSettings.Arguments = $"-c \"make install\"";
+        context.StartProcess(shellCommandPath, processSettings);
+
+        // build libvorbis
+        processSettings.WorkingDirectory = "./vorbis";
+        processSettings.Arguments = $"-c \"make distclean\"";
+        context.StartProcess(shellCommandPath, processSettings);
+        processSettings.Arguments = $"-c \"./autogen.sh\"";
+        context.StartProcess(shellCommandPath, processSettings);
+        processSettings.Arguments = $"-c \"./configure --disable-examples --disable-docs --disable-shared {prefixFlag} {hostFlag}\"";
+        context.StartProcess(shellCommandPath, processSettings);
+        processSettings.Arguments = $"-c \"make -j{Environment.ProcessorCount}\"";
+        context.StartProcess(shellCommandPath, processSettings);
+        processSettings.Arguments = $"-c \"make install\"";
+        context.StartProcess(shellCommandPath, processSettings);
+
+        // build lame
+        processSettings.WorkingDirectory = "./lame";
+        processSettings.Arguments = $"-c \"make distclean\"";
+        context.StartProcess(shellCommandPath, processSettings);
+        processSettings.Arguments = $"-c \"./configure --disable-frontend --disable-decoder --disable-shared {prefixFlag} {hostFlag}\"";
+        context.StartProcess(shellCommandPath, processSettings);
+        processSettings.Arguments = $"-c \"make -j{Environment.ProcessorCount}\"";
+        context.StartProcess(shellCommandPath, processSettings);
+        processSettings.Arguments = $"-c \"make install\"";
+        context.StartProcess(shellCommandPath, processSettings);
+
+        // Build ffmpeg
+        processSettings.WorkingDirectory = "./ffmpeg";
+        processSettings.Arguments = $"-c \"make distclean\"";
+        context.StartProcess(shellCommandPath, processSettings);
+        processSettings.Arguments = $"-c \"./configure {binDirFlag} {configureFlags}\"";
+        context.StartProcess(shellCommandPath, processSettings);
+        processSettings.Arguments = $"-c \"make -j{Environment.ProcessorCount}\"";
+        context.StartProcess(shellCommandPath, processSettings);
+        processSettings.Arguments = $"-c \"make install\"";
+        context.StartProcess(shellCommandPath, processSettings);
     }
 
-    public override void Finally(BuildContext context)
+    private static string GetFFMpegConfigureFlags(BuildContext context)
     {
-        //  Ensure we revert the patched files so when running/testing locally they are put back in original state
-        context.StartProcess("patch", "-R ./buildscripts/vcpkg/ports/ffmpeg/portfile.cmake ./patches/ffmpeg-portfile.patch");
-        context.StartProcess("patch", "-R ./buildscripts/vcpkg/triplets/x64-linux.cmake ./patches/x64-linux-cmake.patch");
+        var ignoreCommentsAndNewLines = (string line) => !line.StartsWith('#') && !line.StartsWith(' ');
```

Comment:
Use `string.IsNullOrWhiteSpace(line)` for the second check instead of checking the starting char.

Reply:
Updated in [04fd7a9](https://github.com/MonoGame/MonoGame.Tool.FFmpeg/pull/3/commits/04fd7a9bbc9d7ffcb559af96328b1053f58c16da)